CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.21.0](https://github.com/redhat-developer/quarkus-ls/milestone/24?closed=1) (December 3, 2024)
+
+### Enhancements
+
+- Data model template matcher support. See [#1021](https://github.com/redhat-developer/quarkus-ls/pull/1021).
+- Proper fix for comments in TemplateExtension quickfix test. See [#1020](https://github.com/redhat-developer/quarkus-ls/pull/1020).

```

Comment:
This is a bug fix

Reply:
@angelozerr once this is addressed I think the changelog will be good.

Reply:
fixed